common/src/main/resources/ffiec_census_2024.txt

```diff
@@ -1,4 +1,4 @@
-CollectionYear|CBSACode|FIPSStateCode|FIPSCountyCode|CensusTract|FFIECMedianFamilyIncome|Population|MinorityPopulationPct|NumOwnerOccupiedUnits|Num1To4FamilyUnits|TractMFI|TractToMSAIncomePct|MedianAge|SmallCounty|MSAOrMDTitle
+Collection Year|MSA/MD|State|County|Census Tract|FFIEC Median Family Income|Population|Minority Population %|Number of Owner Occupied Units|Number of 1 to 4 Family Units|Tract MFI|Tract to MSA Income %|Median Age|Small County|MSA/MD Name
```

Comment:
Is there supposed to be a `ffiec_census_2025.txt` file?

Reply:
Kind of, its  just a renamed 2024 census. We used this method in the past while we wait for the Federal Reserve Board to generate the 2025 census but to avoid confusion we're making sure we see we're using 2024 as a place holder until its ready.